README.md

```diff
@@ -104,12 +104,12 @@ In **PowerShell** or in a **Command Prompt** window:
 
 - **Windows**
 
-    - In PowerShell, upon executing `bundled_installer`, if you see an error with the message "execution of scripts is disabled on this system", set the [execution policy](https://docs.microsoft.com/en-us/powershell/module/microsoft.powershell.security/set-executionpolicy?view=powershell-6) to "RemoteSigned" and then rerun `bundled_installer`.
+    - In PowerShell, if you encounter an error with the message "execution of scripts is disabled on this system", set the [execution policy](https://docs.microsoft.com/en-us/powershell/module/microsoft.powershell.security/set-executionpolicy?view=powershell-6) to `RemoteSigned` and then rerun `bundled_installer`.
       
       ```ps1
       Set-ExecutionPolicy RemoteSigned
       ```
-    - If you encounter an error that states "No module named 'virtualenv'", then you can perform the following commands to remedy this issue:
+    - If you encounter an error with the message "No module named 'virtualenv'", use the following commands to install `virtualenv` and the EB CLI.
```

Comment:
: (colon) at end instead of period.

Reply:
AWS style guidelines call for a period here.